umee-provider-config/endpoints.toml

```diff
@@ -1,4 +1,14 @@
+[[provider_endpoints]]
+name = "osmosis"
+rest = "https://api.osmo-api.prod.ojo.network"
```

Comment:
Looks like this is the default in code: 

https://github.com/ojo-network/price-feeder/blob/9c240bd6790e995b0294b6cfe9c2f5d479e6d2ce/oracle/provider/osmosis.go#L19C21-L19C50